aiogram/dispatcher/storage.py

```diff
@@ -270,29 +273,32 @@ async def reset_bucket(self, *,
         """
         await self.set_data(chat=chat, user=user, data={})
 
-
-class FSMContext:
-    def __init__(self, storage, chat, user):
-        self.storage: BaseStorage = storage
-        self.chat, self.user = self.storage.check_address(chat=chat, user=user)
-
-    def proxy(self):
-        return FSMContextProxy(self)
-
     @staticmethod
-    def _resolve_state(value):
+    def resolve_state(value):
         from .filters.state import State
 
         if value is None:
             return
-        elif isinstance(value, str):
+
+        if isinstance(value, str):
```

Comment:
what's the purpose of creating new comparison branch for each type, when the instance in our case cannot be `State` and `str` at the same time?

Reply:
Since the return statement terminates the execution of the current function, the both forms (if+return/elif+return) are equivalent, but separate checks looks more readable